[2025-05-24T03:33:20.533+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-24T03:33:21.060+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: random_user_pipeline.insert_postgres manual__2025-05-24T03:31:42.290300+00:00 [queued]>
[2025-05-24T03:33:21.086+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: random_user_pipeline.insert_postgres manual__2025-05-24T03:31:42.290300+00:00 [queued]>
[2025-05-24T03:33:21.088+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-05-24T03:33:21.122+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): insert_postgres> on 2025-05-24 03:31:42.290300+00:00
[2025-05-24T03:33:21.133+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=324) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-24T03:33:21.135+0000] {standard_task_runner.py:63} INFO - Started process 325 to run task
[2025-05-24T03:33:21.133+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'random_user_pipeline', 'insert_***', 'manual__2025-05-24T03:31:42.290300+00:00', '--job-id', '447', '--raw', '--subdir', 'DAGS_FOLDER/random_user_pipeline.py', '--cfg-path', '/tmp/tmpan6l2n4x']
[2025-05-24T03:33:21.137+0000] {standard_task_runner.py:91} INFO - Job 447: Subtask insert_***
[2025-05-24T03:33:22.590+0000] {task_command.py:426} INFO - Running <TaskInstance: random_user_pipeline.insert_postgres manual__2025-05-24T03:31:42.290300+00:00 [running]> on host fed8e0f764cf
[2025-05-24T03:33:25.238+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='papala.ramesh@quation.in,smilesoframs@gmail.com,ramesh.rams333@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='random_user_pipeline' AIRFLOW_CTX_TASK_ID='insert_***' AIRFLOW_CTX_EXECUTION_DATE='2025-05-24T03:31:42.290300+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-24T03:31:42.290300+00:00'
[2025-05-24T03:33:25.240+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-24T03:33:28.934+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-24T03:33:28.935+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-24T03:33:29.345+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=random_user_pipeline, task_id=insert_***, run_id=manual__2025-05-24T03:31:42.290300+00:00, execution_date=20250524T033142, start_date=20250524T033321, end_date=20250524T033329
[2025-05-24T03:33:29.393+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-24T03:33:29.818+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py:1297 AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `airflow.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
[2025-05-24T03:33:29.851+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-24T03:33:29.855+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
